scripts/Muon/GUI/Common/contexts/muon_group_pair_context.py

```diff
@@ -336,6 +336,16 @@ def _check_name_unique(self, name):
                 return False
         return True
 
+    def get_group_counts_workspace_names(self, runs, groups, rebin=False):
```

Comment:
can we have an issue to look at if we should use this method in other parts of the code instead of replacing the asymmetry with counts in the name?

Reply:
Yes here it is #31997